src/main/java/com/softserveinc/dokazovi/controller/RegionController.java

```diff
@@ -36,4 +39,12 @@ public ResponseEntity<List<RegionDTO>> getAllRegions() {
 				.status(HttpStatus.OK)
 				.body(regionService.findAllRegions());
 	}
+
+	@PostMapping(REGION_BY_CITY)
```

Comment:
It seems this endpoint doesn't require body to be passed, hence you van use Get mapping.
Or did I miss something?

Reply:
My bad. I thought @PathVariable required @PostMapping too. Fixed